app/_how-tos/kic-install.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Install {{ site.kic_product_name }}
+description: Run {{ site.kic_product_name }} with {{ site.konnect_short_name }} or on-prem using Helm
+content_type: how_to
+permalink: /kubernetes-ingress-controller/install/
+breadcrumbs:
+  - /kubernetes-ingress-controller/
+
+related_resources:
+  - text: All KIC documentation
+    url: /index/kubernetes-ingress-controller/
+
+products:
+  - kic
+
+works_on:
+  - on-prem
+  - konnect
+
+entities: []
+
+tldr:
+  q: How do I install {{ site.kic_product_name }}?
+  a: |
+    ```bash
+    helm install kong kong/ingress -n kong --create-namespace
+    ```
+
+prereqs:
+  skip_product: true
+  expand_accordion: false
+  kubernetes:
+    gateway_api: true
+    gateway_api_optional: true
+
+cleanup:
+  inline:
+    - title: Uninstall KIC from your cluster
+      include_content: cleanup/products/kic
+      icon_url: /assets/icons/kic.svg
+---
+
+{: data-deployment-topology="konnect" }
+## Konnect setup
+
+{:.info}
+> For UI setup instructions to install {{ site.kic_product_name }} on {{ site.konnect_short_name }}, use the [Gateway Manager setup UI](https://cloud.konghq.com/us/gateway-manager/create-control-plane).
+
+To create a {{ site.kic_product_name }} in {{ site.konnect_short_name }} deployment, you need the following items:
+
+1. A [{{ site.konnect_short_name }} access token](https://cloud.konghq.com/global/account/tokens), saved in the `KONNECT_TOKEN` environment variable using `export KONNECT_TOKEN="kpat..."`
+1. A {{ site.kic_product_name }} Control Plane, including the CP URL
+1. An mTLS certificate for {{ site.kic_product_name }} to talk to {{ site.konnect_short_name }}
```

Comment:
I added step 1 to the prereqs, if it makes sense, I would add a prereq the kic CP into the prereqs section as well so you can get export both of those env variables too.

Reply:
Creating a KIC CP is a bit more involved today, and I'd rather not have "run this, then copy and paste these specific values" in a prereq.

Perhaps once the tooling improves we can revisit, but for now I'd like to leave creating a CP as part of the main flow.